src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      calculator.result += number;
+
+      return calculator.result;
+    },
```

Comment:
According comment about `operate` method next code will be:

```suggestion
    add(number) {
      this.result += number;
    },
```

Fix another methods by yourself